### DISCUSSION TITLE
Can we send message by contact object ,not chat object?

### DISCUSSION THREAD

**Discussion by hmbyl** [Auties00/WhatsappWeb4j#87] [General]
Can we send message by contact object ,not chat object?

**Comment by Auties00** [comment]
> Can we send message by contact object ,not chat object?

Well no, you would have to convert the contact to a chat

**Reply by halimselim** [reply]
> convert the contact to a chat

How ?

**Reply by Auties00** [reply]
You can use WhatsappManager#findChatByJid to search for the corresponding chat, and then if it doesn't exists hou can create a place holder with Chat#builder (remember to set at least the required fields)
